src/components/ui/singleBreed/BreedData.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./breedData.scss";
+import { Length } from "../../../types";
+import { SingleBreedSpecsProps } from "../../../types";
+
+function isLength(value: unknown): value is Length {
+  return (
+    (value as Length).metric !== undefined &&
+    (value as Length).imperial !== undefined
+  );
+}
+
+function SpecsItem({ value, name }: { value: unknown; name: string }) {
+  if (typeof value === "string" && value) {
+    return (
+      <p className="breed-data__details__characteristic">
+        <strong>{name.split("_").join(" ")}:</strong> {value}
+      </p>
+    );
+  }
+  if (isLength(value)) {
```

Comment:
why do you check value for isLength and only use value.metric under this condition?

According to your code, If value.imperial will be undefined, {value.metric} Centimeters will not be printed out